packages/ketcher-react/src/script/ui/views/components/StructEditor/StructEditor.jsx

```diff
@@ -200,33 +204,36 @@ class StructEditor extends Component {
         onMouseDown={(event) => event.preventDefault()}
         {...props}
       >
-        <ContextMenuTrigger
+        <FunctionalGroupTrigger
```

Comment:
Hmm, I think the appropriate nesting would be:
```
<ContextMenuTrigger>
  <FunctionalGroupTrigger>
  </FunctionalGroupTrigger>
</ContextMenuTrigger>
```
Or will it break something?

Reply:
Unfortunately, it doesn't work if nesting like that. The `FunctionalGroupTrigger` is directly imported from the library `react-contextmenu`. I guess it captures the event `onContextMenu` and stops it from propagating.
Who nests who doesn't matter, because we are going to refactor `FunctionalGroupTrigger`, like combining it with `ContextMenuTrigger`, right?